### DISCUSSION TITLE
Global filter not working with pagination

### DISCUSSION THREAD

**Discussion by mschuhbauer** [KevinVandy/material-react-table#49] [Q&A] [answered]
The global filter in your [Advanced Example](https://www.material-react-table.com/docs/examples/advanced) isn't working in combination with pagination. The items in the list aren't changing on paginate actions. Is this a Bug or a known incompatibility ?
(Sry im kind of new to your awesome library)

![grafik](https://user-images.githubusercontent.com/11249760/187276632-fa075cdf-ce3a-4602-beb5-1685304c0db9.png)

**Comment by KevinVandy** [comment]
This is a bug that probably popped up recently

**Comment by KevinVandy** [comment, marked as answer]
Fixed this as of `v0.40.8`

**Reply by mschuhbauer** [reply]
Thanks man for this crazy fast answer and bugfix :)
